definitions/npm/redux_v3.x.x/flow_v0.28.x-v0.32.x/redux_v3.x.x.js

```diff
@@ -0,0 +1,54 @@
+declare module 'redux' {
+
+  /*
+
+    S = State
+    A = Action
+
+  */
+
+  declare type Dispatch<A: { type: $Subtype<string> }> = (action: A) => A;
```

Comment:
Looking at this, I'm guessing that making this work with [redux-thunk](https://github.com/gaearon/redux-thunk) which allows action creators to return functions is on the todo list? Or is there a special trick to get that to work?